.claude/settings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "permissions": {
+    "allow": [
+      "WebFetch(domain:mcp.sentry.dev)",
+      "WebFetch(domain:docs.sentry.io)",
+      "WebFetch(domain:develop.sentry.dev)",
+      "WebFetch(domain:modelcontextprotocol.io)",
+      "WebFetch(domain:docs.anthropic.com)",
+      "Bash(grep:*)",
```

Comment:
we could extend these, but id keep them to write only, and trusted domains

its worth noting that any of these commands _do_ have the ability to prompt inject, so we may think this is too high risk, but its also worth noting everyones going to blindly mash yes on these anyways